capability_linux.go

```diff
@@ -25,21 +26,31 @@ const (
 var (
 	capVers    uint32
 	capLastCap Cap
+	once       sync.Once
 )
 
-func init() {
-	var hdr capHeader
-	capget(&hdr, nil)
-	capVers = hdr.version
-
-	if initLastCap() == nil {
-		CAP_LAST_CAP = capLastCap
-		if capLastCap > 31 {
-			capUpperMask = (uint32(1) << (uint(capLastCap) - 31)) - 1
-		} else {
-			capUpperMask = 0
+func getCapMaskCapLast() (uint32, Cap) {
+	once.Do(func() {
+		var hdr capHeader
+		capget(&hdr, nil)
+		capVers = hdr.version
+
+		if initLastCap() == nil {
```

Comment:
I don't think it is a good idea to ignore errors of initLastCap. I think we can return this error from this function.

Reply:
if initLastCap() get error should I throw it ？ there are many func （ mkStringCap  <- StringCap <-mkString<-String()）need to add err to return.
or I write err to std error in GetCapLastCap() func

Reply:
I add getCapMaskCapLast() in newPid to make sure  if initLastCap() will run ok ,we can run next step.